src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1709,9 +1767,7 @@ def add_pool_qty(self, quantity, namespace=None, create=True, connect=True,
         if self.pool is None:
             self.pool = []
         if not isinstance(self.pool, list):
-            self.fail(
-                "add_pool_qty(): self.pool must be a list: {}".format(
-                    type(self.pool)))
+            self.fail("add_pool_qty(): self.pool must be a list: {}".format(type(self.pool)))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string